VAMobile/src/screens/HealthScreen/SecureMessaging/ViewMessage/ViewMessageScreen.tsx

```diff
@@ -405,11 +405,11 @@ function ViewMessageScreen({ route, navigation }: ViewMessageScreenProps) {
           )}
           {replyExpired && (
             <Box my={theme.dimensions.standardMarginBetween}>
-              <AlertBox border={'warning'} title={t('secureMessaging.reply.youCanNoLonger')}>
+              <AlertWithHaptics variant="warning" header={t('secureMessaging.reply.youCanNoLonger')}>
                 <TextView mt={theme.dimensions.standardMarginBetween} variant="MobileBody">
                   {t('secureMessaging.reply.olderThan45Days')}
```

Comment:
Could potentially be simplified moving this to the `description` prop to AlertWithHaptics.